operator_toolkit/soc_helpers/loader.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import pandas
+import arrow
+import cabinets
+from influxdb import DataFrameClient
+
+
+CONFIG = cabinets.read("operator_toolkit/soc_helpers/config.yml")
+
+
+class DataLoader:
+
+    def __init__(self, site, start, end, timezone):
+        """Instantiates the DataLoader class.
+
+        :param site: site_id.
+        :param start: start time, in ISO format.
+        :param end: end time, in ISO format.
```

Comment:
```suggestion
        :param str site: site_id.
        :param str start: start time, in ISO format.
        :param str end: end time, in ISO format.
```
Specifying the `type` in these situations is helpful because there is some ambiguity whether or not this is a string representation of a datetime, or a `datetime` object, or an `Arrow` object, etc.

Reply:
agreed! totally missed this, thanks